src/vivarium_public_health/disease/state.py

```diff
@@ -39,16 +39,26 @@ def setup(self, builder):
 
         self.clock = builder.time.clock()
 
-        columns = [self._model, 'alive']
-        columns += [self.event_time_column, self.event_count_column]
-
-        self.population_view = builder.population.get_view(columns)
-        builder.population.initializes_simulants(self.load_population_columns,
-                                                 creates_columns=[self.event_time_column, self.event_count_column],
+        columns_created = [self.event_time_column, self.event_count_column]
```

Comment:
notes from special disease apply here as well.  You'll find except for a few names, most of the code is identical.